src/bitcoin-tx.cpp

```diff
@@ -273,22 +275,30 @@ static void MutateTxAddOutData(CMutableTransaction& tx, const string& strInput)
 
 static void MutateTxAddOutScript(CMutableTransaction& tx, const string& strInput)
 {
-    // separate VALUE:SCRIPT in string
-    size_t pos = strInput.find(':');
-    if ((pos == string::npos) ||
-        (pos == 0))
+    // separate VALUE:SCRIPT(:SEGWIT)(:P2SH)
+    std::vector<std::string> vStrInput;
+    boost::split(vStrInput, strInput, boost::is_any_of(":"));
```

Comment:
There seems to be a general move away from Boost (https://github.com/bitcoin/bitcoin/projects/3). Sadly, I don't think there's anything in c++11 as simple as `boost::split` for splitting a string (see `RegisterSet()` for another way of splitting the string).

It's probably cleaner to keep this as is, but you may want to consider doing it without boost.